OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -84,6 +84,29 @@ constexpr const char* kLogTimeFormat = "%Y-%m-%dT%H:%M:%E6S";
   } while (0 && (assertion))
 #endif
 
+// Declare the class inside the macro so that file:line are the ones where SCOPED_TIMED_LOG is used.
+#define SCOPED_TIMED_LOG(format, ...)                                                          \
+  class ScopedTimedLog##__LINE__ {                                                             \
+   public:                                                                                     \
+    explicit ScopedTimedLog##__LINE__(std::string&& message)                                   \
+        : message_{std::move(message)}, begin_{std::chrono::steady_clock::now()} {             \
+      LOG("%s started", message_);                                                             \
+    }                                                                                          \
+                                                                                               \
+    ~ScopedTimedLog##__LINE__() {                                                              \
+      auto end = std::chrono::steady_clock::now();                                             \
+      auto duration =                                                                          \
+          std::chrono::duration_cast<std::chrono::duration<double, std::milli>>(end - begin_); \
```

Comment:
`absl::Now` has a different meaning as it gives the wall-clock time, which we want to print. `steady_clock` gives a monotonic time and is exactly made for this use ("This clock is not related to wall clock time [...] and is most suitable for measuring intervals"). It doesn't need to match the profiling code (although it probably does), we are just computing a difference.

Reply:
Well it would be nice if we could take two timestamps in a log for example and subtract them to have an interval and know that the result is coherent with the timed scopes and also with our profiling data. Even if we are just computing a difference, that difference could be different from the value in a profiling scope if we're not using the same clock (if we are cpu throttling for example). What would be the downside of using the same clock across the board?